From 37ff3372829090de524a9d705b498edfeabd4ae9 Mon Sep 17 00:00:00 2001
From: scx <scx.mail@gmail.com>
Date: Sun, 27 Oct 2019 19:41:05 +0100
Subject: [PATCH] Update XDG files

Desktop file:
 - Add Version entry
 - Add TryExec entry
 - Remove Application from Categories (deprecated)
 - Add Keywords entry

AppData file:
 - Update URLs
   - Update homepage URL (http -> https)
   - Add help URL
   - Add donation URL
   - Add translate URL
---
 data/qalculate-gtk.appdata.xml.in | 7 +++++--
 data/qalculate-gtk.desktop.in     | 6 ++++--
 2 files changed, 9 insertions(+), 4 deletions(-)

diff --git a/data/qalculate-gtk.appdata.xml.in b/data/qalculate-gtk.appdata.xml.in
index ddfa649..c67ae24 100644
--- a/data/qalculate-gtk.appdata.xml.in
+++ b/data/qalculate-gtk.appdata.xml.in
@@ -30,10 +30,13 @@
    <caption>The main window with keypad visible</caption>
   </screenshot>
  </screenshots>
- <url type="homepage">http://qalculate.github.io/</url>
+ <url type="homepage">https://qalculate.github.io</url>
  <url type="bugtracker">https://github.com/Qalculate/qalculate-gtk/issues</url>
+ <url type="help">https://qalculate.github.io/manual/index.html</url>
+ <url type="donation">https://www.paypal.me/HannaKnutsson</url>
+ <url type="translate">https://github.com/Qalculate/libqalculate/blob/master/README.translate</url>
  <developer_name>Hanna Knutsson</developer_name>
- <content_rating type="oars-1.0" />
+ <content_rating type="oars-1.0"/>
  <kudos>
   <kudo>HiDpiIcon</kudo>
   <kudo>ModernToolkit</kudo>
diff --git a/data/qalculate-gtk.desktop.in b/data/qalculate-gtk.desktop.in
index 3cf68a2..39bc0a2 100644
--- a/data/qalculate-gtk.desktop.in
+++ b/data/qalculate-gtk.desktop.in
@@ -1,12 +1,14 @@
 [Desktop Entry]
+Version=1.0
 _Name=Qalculate!
 _GenericName=Calculator
 _Comment=Powerful and easy to use calculator
+TryExec=qalculate-gtk
 Exec=qalculate-gtk
 Icon=qalculate
 Terminal=false
 Type=Application
 StartupNotify=true
 StartupWMClass=qalculate-gtk
-Categories=GTK;Application;Utility;Calculator;Science;Math;
-
+Categories=GTK;Utility;Calculator;Science;Math;
+_Keywords=calculation;arithmetic;scientific;financial;
